### DISCUSSION TITLE
Does pyucis offer any tools for merging xml coverage report files?

### DISCUSSION THREAD

**Discussion by nsdjg** [fvutils/pyucis#9] [General]
Hello Matthew,

Does pyucis has a capability for merging two xml files?

John

**Comment by mballance** [comment]
Hi John,
   PyUCIS doesn't currently offer database merge. I started work on this a while back, and just revisited it this past weekend. I'm guessing it's in the couple-week range to complete.

-Matthew

**Reply by nsdjg** [reply]
Hello Matthew,

Thanks for the update.  That is good to know.

John
